guides/common/modules/proc_disabling-dhcp-for-integration.adoc

```diff
@@ -0,0 +1,33 @@
+[id="disabling-dhcp-for-integration_{context}"]
+= Disabling DHCP for integration
+
+// MARC: New module. Content is based on https://github.com/theforeman/foreman-documentation/blob/master/guides/common/modules/proc_disabling-dns-dhcp-tftp-for-unmanaged-networks.adoc, and adjusted for DHCP.
+// Needs to be checked if it is technically correct.
+
+If you want to manually manage a DHCP services, you must prevent {Project} from maintaining this service on the operating system and disable orchestration to avoid errors.
```

Comment:
```suggestion
If you want to manually manage a DHCP service, you must prevent {Project} from maintaining this service on the operating system and disable orchestration to avoid errors.
```